kalite/settings/base.py

```diff
@@ -188,16 +188,15 @@
 # the user running kalite and should be in a user-data
 # storage place.
 
-USER_WRITABLE_LOCALE_DIR = os.path.join(USER_DATA_ROOT, 'locale')
-KALITE_APP_LOCALE_DIR = os.path.join(USER_DATA_ROOT, 'locale')
+from fle_utils.general import ensure_dir
```

Comment:
`fle_utils` is something we would like to get rid of... but don't have time to factor out.. it's just a good idea not to use it anymore.. but below you'll see that it's good to not use `ensure_dir` in this context anyways....

Reply:
Hmm. I didn't know that we will be removing `fle_utils`.. anyway, I used the `ensure_dir` since I get an error if I do this:

```python
def extract_content_pack_metadata(zf, lang):
    lang = lcode_to_django_lang(lang)
    modir = get_po_filepath(lang)

    if not os.path.exists(modir):
        os.mkdir(modir)
```
```
OSError: [Errno 2] No such file or directory: '/Users/me/.kalite/project/locale/en/LC_MESSAGES'
```